app/views/admin/products/index.html.haml

```diff
@@ -12,7 +12,11 @@
 %p= text("views.admin.#{plural_object_name}.index.description")
 
 - if current_user.manager_of?(current_facility)
```

Comment:
Since we're in here, I feel like we should switch to using an ability check. `can?(:create, Product)`

Reply:
Oh, what’s the benefit of that? I thought `current_user.manager_of?` reads very nicely, but perhaps there are some drawbacks I don’t get?

Reply:
Changed in d53a621!